### DISCUSSION TITLE
Zigbee light switch turning off every few minutes

### DISCUSSION THREAD

**Discussion by dbertie1** [arendst/Tasmota#14007] [Q&A] [answered]
I've got a no-neutral three way Zigbee switch connected to a Sonoff Zbbridge.  It's called Kitchen_wall_switch (device number 0xF4FE),

All works fine in terms of pairing, switching on and off etc.  However every few minutes the lights all switch off, for no apparent reason.  Has anyone got any ideas as to the problem?

It seems to turn off at the point that there is a ZbState message - but I can't see why!  Other Zigbee items seem to work fine, am I doing something silly here?

**Output from the Console:**
18:28:23.990 ZIG: {"ZbEZSPReceived":"23000101FEF4DBDFFDFEFF23A46004"}

18:28:23.992 ZIG: {"ZbEZSPReceived":"2400FEF4DBDFFDFEFF23A46000030000"}

18:28:23.996 MQT: tele/zigbridge/RESULT = {"ZbState": "Status":34,"IEEEAddr":"0x60A423FFFEFDDFDB","ShortAddr":"0xF4FE","ParentNetwork":"0x0000","JoinStatus":0,"Decision":3}}

18:28:24.209 ZIG: {"ZbEZSPReceived":"6200DBDFFDFEFF23A460"}

18:28:24.218 MQT: tele/zigbridge/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x60A423FFFEFDDFDB","ShortAddr":"0xF4FE","PowerSource":false,"ReceiveWhenIdle":false,"Security":false}}

18:28:26.337 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Power":0,"0006/4001":0,"0006/4002":0,"0006/8001":1,"0006/8002":0,"Endpoint":1,"LinkQuality":23}}}

18:28:26.454 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Power":0,"0006/4001":0,"0006/4002":0,"Endpoint":2,"LinkQuality":21}}}

18:28:26.471 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Power":0,"Endpoint":1,"LinkQuality":23}}}

18:28:26.960 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Power":0,"0006/4001":0,"0006/4002":0,"Endpoint":3,"LinkQuality":23}}}

18:28:27.163 MQT: tele/zigbridge/Lounge_reading_light/SENSOR = {"ZbReceived":{"Lounge_reading_light":{"Device":"0x2491","Name":"Lounge_reading_light","CT":472,"ColorMode":2,"Endpoint":1,"LinkQuality":28}}}

18:28:28.328 ZIG: Auto-responder: ZbSend {"Device":"0xF4FE","Cluster":"0x000A","Endpoint":1,"Response":"000A/0007":692562508}

18:28:28.337 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Cluster":10,"Read":[7],"ReadNames":{"LocalTime":true},"Endpoint":1,"LinkQuality":21}}}

18:28:32.253 MQT: stat/zigbridge/RESULT = {"ZbSend":"Done"}

18:28:32.957 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Power":1,"Endpoint":3,"LinkQuality":23}}}

18:28:34.716 ZIG: {"ZbEZSPReceived":"9B00DBDFFDFEFF23A46006"}

18:28:40.122 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Power":0,"0006/4001":0,"0006/4002":0,"0006/8001":1,"0006/8002":0,"Endpoint":1,"LinkQuality":28}}}

18:28:40.235 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Power":0,"0006/4001":0,"0006/4002":0,"Endpoint":2,"LinkQuality":21}}}

18:28:40.635 MQT: tele/zigbridge/Kitchen_wall_switch/SENSOR = {"ZbReceived":{"Kitchen_wall_switch":{"Device":"0xF4FE","Name":"Kitchen_wall_switch","Power":1,"0006/4001":0,"0006/4002":0,"Endpoint":3,"LinkQuality":28}}}

18:28:51.091 MQT: tele/zigbridge/Lounge_reading_light/SENSOR = {"ZbReceived":{"Lounge_reading_light":{"Device":"0x2491","Name":"Lounge_reading_light","Power":0,"Endpoint":1,"LinkQuality":31}}}

**Comment by dbertie1** [comment, marked as answer]
Hmm, not sure what is happening - have tried reconnecting a few times, Zbbindstate etc and since then no resets (fingers crossed).

I think the key is:
tele/zigbridge/RESULT = {"ZbState": "Status":34,"IEEEAddr":"0x60A423FFFEFDDFDB","ShortAddr":"0xF4FE","ParentNetwork":"0x0000","JoinStatus":0,"Decision":3}}

Which I have read means that the device is being denied a connection - not sure why, or why it now seems more stable.

**Reply by s-hadinger** [reply]
No it's just the device announcing itself typically when it's joining or rejoining a network. Maybe the device kind of reseted itself periodically. I'm not sure why

**Reply by dbertie1** [reply]
Aha, and thanks - as you can tell I’ve not fully got to grips with Zigbee.  

Still stable, so who knows what has happened!
Many thanks, David
